support/map.c

```diff
@@ -5237,7 +5237,7 @@ Mono_Posix_ToSockaddrIn (struct sockaddr_in *from, struct Mono_Posix_SockaddrIn
 #endif /* ndef HAVE_STRUCT_SOCKADDR_IN */
 
 
-#ifdef HAVE_STRUCT_SOCKADDR_IN6
+#if defined(HAVE_STRUCT_SOCKADDR_IN6) && !defined(HOST_WIN32)
```

Comment:
I missed that, thanks for pointing it out!

If we want to take this approach instead of https://github.com/mono/mono/pull/4754, do I need to modify the create-native-map utility then?